core/python3ActionLoop/requirements.txt

```diff
@@ -0,0 +1,28 @@
+# default available packages for python3action
```

Comment:
synchronize with python 3.9 file.

Reply:
ok syncrhonized but I had to update the versions otherwise they do not work